### ISSUE TITLE
Add support for default, minimum, maximum, minLength, maxLength for properties and params

### ISSUE THREAD

**Issue by paulish** [BRIKEV/express-jsdoc-swagger#181]
OpenAPI supports more attributes for properties and params than it is possible to describe using this library. Some of them are rather important.

For example: `minLength` and `maxLength` are important for string properties when you must limit the length (minimum length for the password field or maximum length due to the database field length limit); `minimum` and `maximum` for integer fields are important too.

Also it would be nice to have an ability to describe `default` values for properties and params.

**Comment by kevinccbsg** [comment]
Hi, @paulish I think it is a good feature. 

We are still dealing with the best way to define this in the comments 🤔

**Comment by paulish** [comment]
Yes. The syntax is the problem and not only for this task. I see some options:
`* @param {integer} value.query.required - value description - options:[maximum: 10, minimum: 0, example: 5]`
or we may use JSON.
`* @param {string} name.query.required - name description - {"maxLength": 30, "example": "Julia"}`

JSON could be more preferable since we don't need to invent another parser and we could pass all possible options to the resulting document.

**Comment by paulish** [comment]
Another important property is `pattern` which can be defined for string properties. It allows to validate strings by regular expressions (very useful for phone numbers, person identifiers like name and surname).

**Comment by YosaRama** [comment]
Hi, this update is really helpful.

but could I have additional value outside the `schema`, because I need add `example` or `default` outside the schema for `parameters` I create.

currently I'm using `dredd` for automation API testing, on OpenAPI v.3 it's need write `example` outside `schema`, is there any possible way to me add some value outside the `schema` ?

Error inside schema (attachment)
![Screenshot 2023-01-20 at 21 09 15](https://user-images.githubusercontent.com/67961793/213702781-d436edff-770b-4f35-b7c2-1c2f32d18c88.png)

Success outside schema (attachment)
![Screenshot 2023-01-20 at 21 10 33](https://user-images.githubusercontent.com/67961793/213702843-23a6d292-e081-45ac-b6b9-18bb18b2ec3e.png)

I already report this problem too on `dredd`, but I think this one can be a good features for this package, it will help for any update on OpenAPI version like create `multiple example for parameters`
![Screenshot 2023-01-20 at 21 14 40](https://user-images.githubusercontent.com/67961793/213703394-ad4d3950-ebf1-47a0-bede-de42c49d55e0.png)

hopefully get help for my problems. Thanks!
